### PR TITLE
Fix support of redirect streams in the frontend

### LINKED ISSUE #1469: Redirect streams cannot be created
[Screencast from 2023-02-13 16-42-12.webm](https://user-images.githubusercontent.com/4013804/218558565-a723016a-27ec-4350-8bcc-4fe3ceb945b9.webm)

### PR DESCRIPTION
**Pull request by Williangalvani**
fix #1469

### CODE REVIEW

**Review by joaoantoniocardoso — COMMENTED** [review]
Great! Just an enhancement for `core/frontend/src/components/video-manager/VideoStreamCreationDialog.vue:24`:

```Diff
<v-select
            v-model="selected_encode"
            :items="available_encodes"
            label="Encoding"
+            :disabled="is_redirect_source"
            :rules="[validate_required_field]"
          />
          <v-select
            v-model="selected_size"
            :items="available_sizes"
            :label="size_selector_label"
-            :disabled="!selected_format"
+            :disabled="!selected_format || is_redirect_source"
            :rules="[validate_not_null]"
          />
          <v-select
            v-model="selected_interval"
            :items="available_framerates"
            :label="framerate_selector_label"
-            :disabled="!selected_size"
+            :disabled="!selected_size || is_redirect_source"
            :rules="[validate_not_null]"
          />
```

**Comment by joaoantoniocardoso** [comment]
Question: should redirect be available in normal mode or just pirate?

File: core/frontend/src/components/video-manager/VideoDevice.vue [hunk 1]
```diff
@@ -115,6 +117,12 @@ export default Vue.extend({
         if ('Local' in stream.video_and_stream.video_source) {
           return stream.video_and_stream.video_source.Local.device_path === this.device.source
         }
+        console.log(stream.video_and_stream.video_source)
```
[the remark this answers is not in the document]
Comment (Williangalvani) [reply core/frontend/src/components/video-manager/VideoDevice.vue]: oops

**Comment by Williangalvani** [comment]
> Great! Just an enhancement for `core/frontend/src/components/video-manager/VideoStreamCreationDialog.vue:24`:
> 
> ```diff
> <v-select
>             v-model="selected_encode"
>             :items="available_encodes"
>             label="Encoding"
> +            :disabled="is_redirect_source"
>             :rules="[validate_required_field]"
>           />
>           <v-select
>             v-model="selected_size"
>             :items="available_sizes"
>             :label="size_selector_label"
> -            :disabled="!selected_format"
> +            :disabled="!selected_format || is_redirect_source"
>             :rules="[validate_not_null]"
>           />
>           <v-select
>             v-model="selected_interval"
>             :items="available_framerates"
>             :label="framerate_selector_label"
> -            :disabled="!selected_size"
> +            :disabled="!selected_size || is_redirect_source"
>             :rules="[validate_not_null]"
>           />
> ```

nice

**Review by joaoantoniocardoso — APPROVED** [review]

**Pull request merged**